web_transport/sdk/impl/web_transport_owt_client_impl.cc

```diff
@@ -150,8 +150,13 @@ void WebTransportOwtClientImpl::OnError(const net::WebTransportError& error) {
 }
 
 void WebTransportOwtClientImpl::OnClosed(
-      const absl::optional<net::WebTransportCloseInfo>& close_info) {
-  LOG(INFO) << "OnClosed.";
+    const absl::optional<net::WebTransportCloseInfo>& close_info) {
+  if (!visitor_) {
```

Comment:
one common comment. consider adding calling thread checker for those callbacks?

Reply:
no need for the brackets here.

Reply:
This method is called by `net::WebTransportClient`. It's supposed to be called on correct thread. We can add a thread checker for these methods after threading model change (described in PR #60).